### PR TITLE
Add example of theme change

### PR DESCRIPTION
**Pull request by rafaelovandov**
**Description**
Inspired by responsive builder pr, I thought of idea of how to have themes in the app without code anything for colors, etc.

**Demo**
https://github.com/FilledStacks/expression-ui/assets/66031839/f7e5e27b-cef2-48a0-ad11-71d0c5659a56

### CODE REVIEW

**Comment by rafaelovandov** [comment]
I think the best approach to using theme and responsive together is to check if the artboard name contains light or dark and mobile, tablet, or desktop

**Review by FilledStacks — DISMISSED** [review]
Ooooooooooh nice. 

I like this idea 💡 Very awesome, thank you @rafaelovandov

File: examples/example-flutter/lib/ui/views/startup/startup_view.dart [hunk 1]
```diff
@@ -15,7 +15,10 @@ class StartupView extends StackedView<StartupViewModel> {
   ) {
     return Scaffold(
       body: ExpressionView(
-        artboardName: 'intro-view',
+        artboardName:
+            MediaQuery.of(context).platformBrightness == Brightness.light
```
Comment (FilledStacks) [inline examples/example-flutter/lib/ui/views/startup/startup_view.dart]: This removes the Startup view intro. 

Can you please instead create a new view called themeView and add these changes in there. 

I'll link them all together in the next update of the example app.

File: examples/example-flutter/lib/constants/expression_constants.dart [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const String ksExpressionUIFilePath = 'assets/expressionui.riv';
+const String ksExpressionUIFilePath = 'assets/app.riv';
```
Comment (FilledStacks) [inline examples/example-flutter/lib/constants/expression_constants.dart]: Please add a new value instead for your new .riv file

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Awesome work, I really like the idea 💡

I appreciate you taking the effort to make this PR.

**Comment by rafaelovandov** [comment]
I created the new theme view, you need only to update the artboard names and add new views in your riv. 
Do you have a Discord server or something to communicate faster than this? This idea is pretty cool. I have some questions.

**Comment by FilledStacks** [comment]
@rafaelovandov yes I do, I just got back from paternity leave. 

Here's a link to the discord. https://discord.gg/FbaVQPPc

**Review by FilledStacks — APPROVED** [review]

**Pull request merged**